doc/ipc.adoc

```diff
@@ -272,7 +272,7 @@ A concrete mapping of these operations to IPCs might go as follows.
 
 - Message is a four-byte struct containing `fd` as a little-endian `u32`. Borrow
   0 is `buffer` and must be writable.
-- Data will be written to a prefix of borrow 0, starting at offset 0.
+- Data will be written to a prefix of lease 0, starting at offset 0.
```

Comment:
Both are technically correct here, since "borrow" is the term used at the server side, while "lease" is from the client's perspective. I'm happy to accept this change though since this section is a little informal, and the old usage clearly surprised you.